docs/src/catalyst_functionality/network_analysis.md

```diff
@@ -52,7 +52,7 @@ the reaction rate equation ODE model for the repressilator is
 ## Matrix-vector reaction rate equation representation
 In general, reaction rate equation (RRE) ODE models for chemical reaction networks can
 be represented as a first-order system of ODEs in a compact matrix-vector notation. Suppose
-we have a reaction network with ``K`` reactions and ``M`` species, labelled by the state vector
+we have a reaction network with ``K`` reactions and ``M`` species, labelled by the unknown vector
```

Comment:
```suggestion
we have a reaction network with ``K`` reactions and ``M`` species, labelled by the state vector
```
And fix anywhere else this PR is reintroducing this.

Reply:
fixed these